### DISCUSSION TITLE
User and install from Store without account

### DISCUSSION THREAD

**Discussion by KnifMelti** [Romanitho/Winget-Install-GUI#53] [Ideas]
```
if ($null -ne ${env:=::}) {
    # Session is not Elevated
    "--source msstore, --scope user and skip Get-WingetStatus/notify if not installed"
}
```
Compile **WiGui** without **-requireAdmin**.

Could it be possible?

**Comment by KnifMelti** [comment]
Store for Business/Education is closing.

**Comment by Romanitho** [comment]
Yes, you can compile WiGui without admin right. You'll have to remove the "run as" as well when Winget-install and WAU are called.

then, you will be able add --scope in the applist
